src/js/App/Header/UserToggle.js

```diff
@@ -20,9 +23,14 @@ function buildItems(username, isOrgAdmin, accountNumber = -1, isInternal, extraI
     </DropdownItem>,
     <React.Fragment key="account wrapper">
       {accountNumber > -1 && (
-        <DropdownItem key="Account" isDisabled>
```

Comment:
This will cause issues with a cursor. See the attached recording:
![Peek 2022-08-04 10-41](https://user-images.githubusercontent.com/22619452/182803934-2fb10021-dd1c-4044-b45e-6fa365e9ba5a.gif)

I get that the tooltip does not work if the item is disabled. But we have to change the styling if the child elements to fix the cursor.

Reply:
I've fixed this issue in the latest commit. I used mouse-events. I'm not sure if there's a cleaner way. If there is, please let me know.